src/status_im/transport/shh.cljs

```diff
@@ -60,15 +60,16 @@
 
 (re-frame/reg-fx
  :shh/post
- (fn [{:keys [web3 message success-event error-event]
-       :or {error-event :protocol/send-status-message-error}}]
-   (post-message {:web3            web3
-                  :whisper-message (update message :payload (comp transport.utils/from-utf8
-                                                                  transit/serialize))
-                  :on-success      (if success-event
-                                     #(re-frame/dispatch (conj success-event %))
-                                     #(log/debug :shh/post-success))
-                  :on-error        #(re-frame/dispatch [error-event %])})))
+ (fn [post-calls]
```

Comment:
Are we sure we always get an array of post-request? or it happens only if we use `merge-fx`?

Reply:
We are, I search for all occurrences of `:shh/post` is project and changed it to vector (array) instead of the single call. Luckily, it's only ever used in `send`/`send-with-pubkey` functions, so there are only 2 such places :)

Reply:
Cool, tests are failing (seen ids), feel free to disable for now if it's in rush to get to QA